errai-common-apt/src/main/java/org/jboss/errai/common/apt/ErraiAptExportedTypes.java

```diff
@@ -97,9 +95,11 @@ private String getAnnotationNameFromExportFileElement(final Element e) {
   }
 
   private void addLocalExportableTypesWhichHaveNotBeenExported() {
+    final long start = System.currentTimeMillis();
     System.out.println("Exporting local exportable types..");
     exportedAnnotationsPackageElement(elements).map(this::getAllExportedAnnotations)
             .ifPresent(this::addAllExportableTypes);
+    System.out.println("Exported local exportable types in " + (System.currentTimeMillis() - start) + "ms");
```

Comment:
Do you mean remove all `System.out.println`s or just this one? There's something called `Messager` in the APT environment we could use. I tried to, but it doesn't show during the build and I didn't dig much into that to find out why. Do you want that to be part of this PR?

Reply:
IIRC the Messager is for raising errors or giving feedback to the developer using the annotation processor (i.e. showing compile errors in an IDE). Eventually we may want to retrofit some of this into our annotation processors (instead of just throwing exceptions).

But for messages like this one that act like debug logging I think we want to use some kind of logger.

Reply:
Oh, okay. I was treating the `System.out` as a `log.info`. I think we should discuss we should log as `INFO` during the build. I think it's good to provide a feedback of which classes are being generated, because when we have automatic reload, it's going to be almost necessary for us to see which classes are being regenerated. I understand `System.out` is far from ideal, but maybe using the `Messager` to print some `"Generating class Foo"` `"Generated class Foo in Xms."` might be a good idea.

Do you want me to address that in this PR or maybe in a future one? I'm positive we won't call this branch ready to merge before we've done it :)

Reply:
It looks like there are more System.out calls in previous code you added that I didn't notice before, so probably we should do a single pass change these later.